src/graphql.rs

```diff
@@ -65,6 +65,7 @@ pub enum GraphQLError {
     ServerError,
     UnauthorizedError(Resource, Operation),
     ValidationError(String, String),
+    ProjectNotFoundError(String),
```

Comment:
I don't know if there's a way to enforce this in Clippy, but the enum variants are sorted.

Reply:
I didn't see any support in clippy for catching this, but I did fix it.